apps/dashboard/src/hooks/use-activities.ts

```diff
@@ -0,0 +1,38 @@
+import { useQuery } from '@tanstack/react-query';
```

Comment:
Please rename to `use-fetch-activities.ts` to align with the hook naming conventions.